### DISCUSSION TITLE
Getting white space issue on the left and right side of the screen with my code.I have tried setting up the "w-screen" for the parent container.Still no breakthrough

### DISCUSSION THREAD

**Discussion by srinivasch135** [tailwindlabs/tailwindcss#11519] [Help]
Here is my code about the issue.

```
import React from 'react';
import {Link} from 'react-router-dom'
// import jobtracker from '../assets/Images/jobtracker.svg';
import jobtracker from '../assets/Images/jobtracker.jpg';

const Home = () => {
  return (
    <div className=' mx-auto p-6 min-h-screen w-full overflow-x-hidden grid bg-slate-100 font-serif '>
      <nav className='text-blue-500 font-extrabold text-2xl'>
         JobTracker
      </nav>
      <div className='flex flex-col   items-center  sm:grid grid-cols-2 '>
        <div className='max-w-xl space-y-4'>
           <h1 className='text-5xl'>Job<span className='text-blue-300'>Trackr</span></h1>
           <p className='text-xl'>With JobTracker, say goodbye to the hassle of managing multiple spreadsheets or notes. Focus on what
           truly matters - landing your ideal job.Experience the efficiency, convenience, and confidence that JobTracker brings to your job search journey.</p>
           <div>
           <Link to="/register" type="button" className='bg-blue-400 rounded-md inline-block p-3 px-4 text-blue-50'>
            Login/Register
           </Link>
           </div>
        </div>
        <div>
          <img src={jobtracker} alt=""  className=' hidden  w-9/12 h-9/12 sm:block ' />
          {/* <jobtracker/> */}
        </div>
      </div>

    </div>
  )
}

export default Home
```

**Comment by wongjn** [comment]
Could you elaborate more regarding the white space "issue" on the left and right side of the screen? What did you *want* to happen, compared to what *is* happening?

**Reply by srinivasch135** [reply]
Actually it's a home page of an application.Let me elaborate bit more clear below.

- On the top left corner i have the site logo which is JobTracker
- In the center of screen vertically i have defined a grid layout in the desktop layout with 2 columns where as in the mobile layout i customized it to flex where the content can be aligned one after the other using flex and flex direction as column.
- In the mobile layout for few types of mobile screens with a different dimensions.I can see some white space is left out either on the left or right side of the screen.

I'm inserting the exact issue snapshot so that you can understand more clearly.
![image](https://github.com/tailwindlabs/tailwindcss/assets/60247258/3805b882-49e5-4143-982c-ff5b9a920a24)

You can see the white space on the right side of layout where as entire layout should be in bg-slate-100

**Reply by wongjn** [reply]
Could it be subpixel rendering on your screen? I see that you have the viewport zoomed out at 72% so subpixel rendering could occur on a low-pixel-density screen. I have replicated your DOM on [Tailwind Play](https://play.tailwindcss.com/d8qwHQ7Ruy?size=412x914) and was unable to see the unexpected behavior.

Consider providing a [Tailwind Play](https://play.tailwindcss.com/) that reproduces the unexpected behavior.

**Reply by srinivasch135** [reply]
Only for few mobile screens i can see this issue.

**Reply by wongjn** [reply]
Could it be subpixel rendering on your screen? I see that you have the viewport zoomed out at 72% so subpixel rendering could occur on a low-pixel-density screen. I have replicated your DOM on [Tailwind Play](https://play.tailwindcss.com/d8qwHQ7Ruy?size=412x914) and was unable to see the unexpected behavior.

Consider providing a [Tailwind Play](https://play.tailwindcss.com/) that reproduces the unexpected behavior.

**Reply by srinivasch135** [reply]
it can be good even at 75%.How can we set the custom percentages in tailwind play to provide you there?

**Reply by Burak-Kumas** [reply]
> Could you elaborate more regarding the white space "issue" on the left and right side of the screen? What did you _want_ to happen, compared to what _is_ happening?

You can simply add the following code globally to the base.css file. Also, try applying overflow-x: hidden; to the html and body tags.


```
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base{
html,  body{
    @apply overflow-x-hidden;
  }
}
